client/src/logger/types.ts

```diff
@@ -0,0 +1,43 @@
+export enum Service {
+    CLIENT = 'client',
+    SERVER = 'server'
+}
```

Comment:
לא אמור להיות בתיקיית enums?

Reply:
technically yes but those enums are specially for the logs, I'll ask the team